src/components/pages/creation.tsx

```diff
@@ -1,5 +1,99 @@
 function Creation(): JSX.Element {
-  return <div>Creation</div>;
+  return (
+    <div style={{ textAlign: 'left', paddingLeft: '2.5vw' }}>
+      Creation and Deletion
```

Comment:
This is a title, so use the < h1> tag

Reply:
I have added it but did not commit for the same reason as below